hermann.gemspec

```diff
@@ -31,13 +31,9 @@ Gem::Specification.new do |s|
     # dependency of your gem. if it is a runtime dependencies the require_jars
     # file will be overwritten during installation.
     s.add_dependency 'jar-dependencies', '~>0.1.2'
-    s.requirements << "jar org.apache.kafka:kafka_2.10, 0.8.1.1"
-    s.requirements << "jar org.mod4j.org.eclipse.xtext:log4j, 1.2.15"
-    s.requirements << "jar org.scala-lang:scala-library, 2.10.1"
-    s.requirements << "jar com.yammer.metrics:metrics-core, 2.2.0"
-    s.requirements << "jar org.slf4j:slf4j-api, 1.7.2"
-    s.requirements << "jar log4j:log4j, 1.2.17"
-    s.requirements << "jar com.101tec:zkclient, 0.3"
+    s.requirements << "jar org.apache.kafka:kafka_2.10, ~>0.8.1.1"
+    s.requirements << "jar org.apache.curator:curator-framework, ~>2.7"
```

Comment:
Curator was in the Jarfile more to allow for local testing using Curator. Curator isn't actually a requirement for Hermann to be run or used of the JVM, but rather we will use it for discovery if it's available.

I'm not against making it a dependency since it's a damned good library, but I'm not sure if this recent of a dependency requirement will break incorporation of Hermann into Storm or other JVM-based environments which already have Zookeeper libraries of some form loaded.